include/mlir/IR/Builders.h

```diff
@@ -148,6 +148,8 @@ class Builder {
                          ArrayRef<AffineExpr> results);
 
   // Special cases of affine maps and integer sets
+  /// Returns a zero result affine map with no dimensions or symbols: () -> ().
+  AffineMap getNullAffineMap(MLIRContext *context);
```

Comment:
This sounds like it would return an AffineMap wrapping a null pointer (which is what AffineMap() gives you).  Please consider something like getEmptyAffineMap or getZeroOutputAffineMap instead.